alica_engine/src/engine/BasicPlan.cpp

```diff
@@ -0,0 +1,10 @@
+#include "engine/BasicPlan.h"
+
+namespace alica {
+
+BasicPlan::BasicPlan(const std::string &name)
+        : _name(name)
```

Comment:
```suggestion
BasicPlan::BasicPlan(std::string name)
        : _name(std::move(name))
```
Also, your formatter is not using the `.clang-format` file.